### PR TITLE
Jekyll 3 support

### PR DESCRIPTION
**Pull request by dmitryzuev**
Updated version of `jekyll-slim` gem, now using new version of `sliq`.

If someone provide `.slim` files with liquid tags for testing, it will be great

### CODE REVIEW

**Comment by minad** [comment]
@dmitryzuev I also added you to this repository and gem. Thx for your efforts!

**Comment by dmitryzuev** [comment]
@minad thanks :)
I'll merge this when I'll be sure that it works well

**Comment by socialphotos** [comment]
@dmitryzuev have you had time to review it?

**Comment by karir-workstation** [comment]
Any clue when will this updates be merged?

**Comment by socialphotos** [comment]
@karir-workstation to use this branch add this code to jekyll Gemfile
```
group :jekyll_plugins do
  gem 'sliq', github: 'dmitryzuev/sliq', branch: 'feature-versions-update'
  gem 'jekyll-slim', :github => 'dmitryzuev/jekyll-slim', :branch => 'feature-jekyll3'
end
```

File: lib/jekyll/slim/version.rb
Comment (seuros) [inline lib/jekyll/slim/version.rb]: this should be bumper to a bigger version , maybe 0.20.0

File: jekyll-slim.gemspec
Comment (seuros) [inline jekyll-slim.gemspec]: debug gem should be removed

File: testapp/.gitignore
Comment (seuros) [inline testapp/.gitignore]: we can remove the testapp.

**Review by seuros — CHANGES_REQUESTED** [review]
I tested these change, and they work perfectly.

Let some comments

**Comment by brandonsturgeon** [comment]
Hey @socialphotos , I'm getting non-https warnings with your snippet there. I can't quite figure out how to reference a specific branch with a url, though. Any thoughts?

File: lib/jekyll/slim/version.rb
Comment (rlue) [inline lib/jekyll/slim/version.rb]: From the [Semantic Versioning specification](http://semver.org/):

> A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes. X is the major version, Y is the minor version, and Z is the patch version. **Each element MUST increase numerically. For instance: 1.9.0 -> 1.10.0 -> 1.11.0.**

File: jekyll-slim.gemspec
Comment (rlue) [inline jekyll-slim.gemspec]: debug gem should be ~~removed~~ listed with `add_development_dependency` instead. (Bundler inherits dependencies from the gemspec, and if byebug isn't specified, it's going to break someone's development workflow.)

**Comment by donv** [comment]
@dmitryzuev @minad Could you merge this?  I'd like to contribute, and it would be easier if I could work against master.

**Comment by donv** [comment]
@minad Any chance you could merge this?

**Comment by minad** [comment]
@donv do you want to maintain this library?

**Comment by seuros** [comment]
@minad my branch is the most up to date one.

**Comment by minad** [comment]
@seuros I can also add you if you want. what is missing here is a maintainer...

**Comment by seuros** [comment]
Sure thing.

**Comment by donv** [comment]
@minad Yes, I can maintain this library for the coming year at least.

**Comment by minad** [comment]
Ok, I invited you both

**Comment by donv** [comment]
Joined.  Thanks!

**Comment by d-Pixie** [comment]
@minad @donv @seuros How come this is still not fixed/merged? 🙂

**Comment by minad** [comment]
@d-Pixie Since no one is maintaining this package currently.

**Review by seuros — APPROVED** [review]

**Pull request merged**
